### ISSUE TITLE
Static build submit file requests too much RAM

### ISSUE THREAD

**Issue by duncan-brown** [ligo-cbc/pycbc#1043]
The static build file requests 32Gb of RAM which is overkill:

https://github.com/ligo-cbc/pycbc/blob/master/tools/static/build_dag.sh#L12

Asking for 500 Mb seems to be reasonable, as PyInstaller only uses a few hundred Mb in the build process:
```
KiB Mem:   4059344 total,  3805964 used,   253380 free,     5828 buffers
KiB Swap:  4192252 total,   148156 used,  4044096 free.  1544328 cached Mem

  PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND
10846 soumi.de  30  10  883004 115060  23712 D  65.2  2.8   0:29.28 pyinstaller
```

**Comment by duncan-brown** [comment]
@soumide1102 if your local universe build on sugar-test1 works OK, you might as well change the universe to ``local`` on line 10:

https://github.com/ligo-cbc/pycbc/blob/master/tools/static/build_dag.sh#L10

**Comment by soumide1102** [comment]
Fixed by #1042.
